templates/main.html

```diff
@@ -209,7 +209,8 @@ <h3>
     </div>
     <footer>
       <small data-i18n="gci-copyright">Google Code-in and the Google Code-in logo are trademarks of Google Inc.</small><br />
-      <small><span data-i18n="gitlab-copyright-1">The GitLab logo is licensed under</span> <a href="https://gitlab.com/gitlab-com/gitlab-artwork/blob/master/LICENSE">CC BY-NC-SA 4.0</a> <span data-i18n="gitlab-copyright-2">by GitLab Inc.</span></small>
+      <small><span data-i18n="gitlab-copyright-1">The GitLab logo is licensed under</span> <a href="https://gitlab.com/gitlab-com/gitlab-artwork/blob/master/LICENSE">CC BY-NC-SA 4.0</a> <span data-i18n="gitlab-copyright-2">by GitLab Inc.</span></small><br/>
```

Comment:
Add a space before the `/` in your break, like `<br />`

Reply:
issues with rebase as always.. it is picking previous commit (first) and latest commit(second) in the text editor. do I need to remove the previous commit reference (d09afc4)?